File: app/src/main/java/com/jeketos/associatewith/guesser/chat/ChatItem.java
package com.jeketos.associatewith.guesser.chat;

/**
 * Created by eugene.kotsogub on 11/7/16.
 *
 */

public class ChatItem implements IChatItem {

    public static final String NAME = "name";
    public static final String MESSAGE = "message";

    private String name;
    private String message;

    @Override
    public String getMessage() {
        return message;
    }

    @Override
    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String getName() {
        return name;
    }

    @Override
    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/com/jeketos/associatewith/guesser/chat/ChatItem.kt
package com.jeketos.associatewith.guesser.chat

/**
 * Created by eugene.kotsogub on 11/7/16.
 *
 */

class ChatItem() : IChatItem {

    lateinit private var  name : String
    lateinit private var  message : String

    override fun getMessage(): String {
        return message
    }

    override fun setMessage(message: String) {
        this.message = message
    }

    override fun getName(): String {
        return name
    }

    override fun setName(name: String) {
       this.name = name
    }
}
